pkg/apis/org/v1/che_types.go

```diff
@@ -296,180 +260,157 @@ type CheClusterSpecServer struct {
-	// Name of a Identity provider (Keycloak / RH SSO) `client-id` that should be used for Che.
+	// Name of a Identity provider (Keycloak / RH SSO) `client-id` that is used for Che.
 	// This is useful to override it ONLY if you use an external Identity Provider (see the `externalIdentityProvider` field).
-	// If omitted or left blank, it will be set to the value of the `flavour` field suffixed with `-public`.
+	// When omitted or left blank, it is set to the value of the `flavour` field suffixed with `-public`.
 	// +optional
 	IdentityProviderClientId string `json:"identityProviderClientId,omitempty"`
 	// Password for The Identity Provider (Keycloak / RH SSO) to connect to the database.
 	// This is useful to override it ONLY if you use an external Identity Provider (see the `externalIdentityProvider` field).
-	// If omitted or left blank, it will be set to an auto-generated password.
+	// When omitted or left blank, it is set to an auto-generated password.
 	// +optional
 	IdentityProviderPostgresPassword string `json:"identityProviderPostgresPassword,omitempty"`
 	// The secret that contains `password` for The Identity Provider (Keycloak / RH SSO) to connect to the database.
-	// If the secret is defined then `identityProviderPostgresPassword` will be ignored.
-	// If the value is omitted or left blank then there are two scenarios:
+	// When the secret is defined, the `identityProviderPostgresPassword` is ignored. When the value is omitted or left blank, the one of following scenarios applies:
 	// 1. `identityProviderPostgresPassword` is defined, then it will be used to connect to the database.
-	// 2. `identityProviderPostgresPassword` is not defined, then a new secret with the name `che-identity-postgres-secret`
-	// will be created with an auto-generated value for `password`.
+	// 2. `identityProviderPostgresPassword` is not defined, then a new secret with the name `che-identity-postgres-secret` will be created with an auto-generated value for `password`.
 	// +optional
 	IdentityProviderPostgresSecret string `json:"identityProviderPostgresSecret,omitempty"`
 	// Forces the default `admin` Che user to update password on first login. Defaults to `false`.
 	// +optional
 	UpdateAdminPassword bool `json:"updateAdminPassword"`
 	// Enables the integration of the identity provider (Keycloak / RHSSO) with OpenShift OAuth.
-	// Empty value on the OpenShift platform by default.
-	// If user changes this empty value to true/false, then che-operator respect this value.
-	// Otherwise che-operator tries to auto detect if Openshift oAuth can be enabled and change empty value,
-	// correspondly to auto-detection result.
-	// This property allows users to directly login with their Openshift user through the Openshift login,
-	// and have their workspaces created under personal OpenShift namespaces.
+	// Enabled by default on OpenShift. This will allow users to directly login with their OpenShift user through the OpenShift login,
```

Comment:
I think we have mecanism to auto-enable oauth. By default property is an empty.

Reply:
Agree